guides/v3.4.0/tutorial/service.md

```diff
@@ -197,7 +197,11 @@ export default Component.extend({
   didInsertElement() {
     this._super(...arguments);
     let mapElement = this.maps.getMapElement(this.location);
-    this.$('.map-container').append(mapElement);
+    
+    let componentElement = document.getElementById(this.elementId);
```

Comment:
I think you can do `this.element` instead and it will have the same effect.

Reply:
Ah! Learning something new every day :) Fixed